### PR TITLE
upgrade npd image version to v0.8.13 for kubemark

### PR DESCRIPTION
**Pull request by tukwila**
#### What type of PR is this?
/kind cleanup

#### What this PR does / why we need it:
refer to the PR: https://github.com/kubernetes/kubernetes/pull/118909
kubemark should upgrade node-problem-detector version to  v0.8.13 as well.

#### Which issue(s) this PR fixes:

Fixes #

#### Does this PR introduce a user-facing change?

```release-note

```

#### Additional documentation e.g., KEPs (Kubernetes Enhancement Proposals), usage docs, etc.:

```docs

```

### CODE REVIEW

File: build/dependencies.yaml [hunk 1]
```diff
@@ -90,7 +90,9 @@ dependencies:
     version: 0.8.13
     refPaths:
     - path: test/e2e_node/image_list.go
-      match: const defaultImage
+       match: const defaultImage
+    - path: test/kubemark/resources/hollow-node_template.yaml
+      match: registry.k8s.io/node-problem-detector/node-problem-detector:v0.8.13
```
Comment (dims) [inline build/dependencies.yaml:95]: remove `:v0.8.13` from this line

Comment (pacoxu) [inline build/dependencies.yaml:95]:
```suggestion
      match: registry.k8s.io/node-problem-detector/node-problem-detector
```

File: test/kubemark/resources/hollow-node_template.yaml [hunk 2]
```diff
@@ -125,8 +125,6 @@ spec:
           requests:
             cpu: {{npd_millicpu}}m
             memory: {{npd_mem_Ki}}Ki
-        securityContext:
-          privileged: true
```
Comment (pacoxu) [inline test/kubemark/resources/hollow-node_template.yaml:129]: why remove this?

/lgtm cancel

**Pull request merged**